src/RA_AchievementSet.cpp

```diff
@@ -421,38 +421,43 @@ BOOL AchievementSet::LoadFromFile(GameID nGameID)
     const std::string sFilename = GetAchievementSetFilename(nGameID);
 
     SetCurrentDirectory(NativeStr(g_sHomeDir).c_str());
-    FILE* pFile = nullptr;
-    errno_t nErr = fopen_s(&pFile, sFilename.c_str(), "r");
-    if (pFile != nullptr)
+
+    std::ios::sync_with_stdio(false);
+
+    if (std::ifstream ifile{ sFilename }; !ifile.is_open())
+    {
+        //	Cannot open file
+        RA_LOG("Cannot open file %s\n", sFilename.c_str());
+        RA_LOG("Error %s\n", ra::GetLastErrorMsg());
```

Comment:
Windows kept on telling me me "Cannot create a file that already exists". This had no affect on core achievements, once I put a local achievement in it stopped giving an error.

Reply:
ifstream will set errno, not GetLastErrror, the "file already exists" error must be from something else.

Reply:
also, `%s` requires you `.c_str()` the `tstring`.